README.md

```diff
@@ -6,9 +6,9 @@ For more details read the [Community Committee Charter](https://github.com/nodej
 
 ## Why does the Community Committee exist?
 
-The Node.js project has had its fair share of problems regarding diversity, and its fair share of attempts to fix that.
+This Community Committee reflects a formal role and the relevance of the voice of community in the governance of the Node.js project. The formation of this group as a top-level committee alongside the Technical Steering Committee(TSC) is communicated value of the contributors of Node.js that roles other than those of code contributor help foster a healthy, sustainable open source community.
```

Comment:
I like the changes generally - I think that this is more meaningful with "The Node.js Project" since inclusivity should be something _the whole project_ stands for.